next.config.js

```diff
@@ -1,3 +1,9 @@
+const baseUrl = process.env.VERCEL_ENV
+  ? `https://${process.env.VERCEL_URL}`
+  : 'http://localhost:3000'
+
+console.log(`BASE_URL: ${baseUrl}`)
```

Comment:
Console log

Reply:
Meant to comment this, I left this in so it's clear what this env var is in build logs and locally.